main.py

```diff
@@ -0,0 +1,27 @@
+print("Loading KiliAutoML...")
```

Comment:
Not useful

Reply:
in fact if can be, there is some loading time. So you can use `kili_print("Loading Kili AutoML...)`

Reply:
Loading all the libraries (pytorch, cv2, pandas...) takes 8 seconds on the mac before anything appears on the terminal when doing kiliautoml --help

Reply:
This highlights an issue imho. We should probably load the modules in a more dynamic fashion (we do not use everything all the time).